### PR TITLE
Testing - Put help methods in help classes

### PR DESCRIPTION
**Pull request by jwigert**

### CODE REVIEW

File: clean-abap/CleanABAP.md [hunk 1]
```diff
@@ -4163,6 +4164,38 @@ class hiring_test defintion
...
+```abap
+CLASS lth_unit_tests DEFINITION ABSTRACT FOR TESTING
+  DURATION SHORT
+  RISK LEVEL HARMLESS.
+
+  PROTECTED SECTION.
+    CLASS-METHODS assert_activity_entity
+      IMPORTING
+        !actual_activity_entity TYPE REF TO zcl_activity_entity
+        !expected_activity_entity TYPE REF TO zcl_activity_entity.
```
Comment (thomham) [inline clean-abap/CleanABAP.md]: no ! please

File: clean-abap/CleanABAP.md [hunk 2]
```diff
@@ -4163,6 +4164,38 @@ class hiring_test defintion
 endclass.
 ```
 
+#### Put help methods in help classes
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Testing](#testing) > [Test Classes](#test-classes) > [This section](#put-help-methods-in-help-classes)
+
+Put help methods used by several test classes in an abstract help class.
```
Comment (thomham) [inline clean-abap/CleanABAP.md]: I would not limit help methods as abstract parent classes. This may lead to the habit to have unrelated code in the helper class, instead of having dedicated helper classes for them with delegation.

File: clean-abap/CleanABAP.md (hunk 1 above)
Comment (jwigert) [reply clean-abap/CleanABAP.md]: Thanks.

File: clean-abap/CleanABAP.md (hunk 2 above)
Comment (jwigert) [reply clean-abap/CleanABAP.md]: Thanks, that is a valid point. I rewrote it a bit. Please let me know your thoughts.
